README.md

```diff
@@ -1080,3 +1080,15 @@ Please join our friendly [GitHub discussions](https://github.com/wakujs/waku/dis
 ## Roadmap
 
 Waku is in active development and we’re seeking additional contributors. Check out our [roadmap](https://github.com/wakujs/waku/issues/24) for more information.
+
+## Contributing
```

Comment:
Actually, I misunderstood. This was added to README.md. It should have added to CONTRIBUTING.md.

@jonathanhefner Can you move this section from README.md to CONTRIBUTING.md please? Feel free to improve CONTRIBUTING.md too.